web/src/main/java/org/devgateway/ocds/web/rest/controller/excelchart/CostEffectivenessExcelController.java

```diff
@@ -41,7 +41,7 @@
     @RequestMapping(value = "/api/ocds/costEffectivenessExcelChart", method = {RequestMethod.GET, RequestMethod.POST})
     public void costEffectivenessExcelChart(@ModelAttribute @Valid final GroupingFilterPagingRequest filter,
                                             final HttpServletResponse response) throws IOException {
-        final String chartTitle = "cost effectiveness";
+        final String chartTitle = "Cost effectiveness";
```

Comment:
@ionutdobre what do u think we do an endpoint to provide some metainformation about charts, that can be reusable both in backend and frontend, like

- chart  title
- description
- legend
- more crazy stuff

the interface is i18n. i am afraid the next thing they will ask you do is translate it in Vietnamese.  😱